Rubberduck.Interaction/IMessageBox.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Forms = System.Windows.Forms;
+
+namespace Rubberduck.Interaction
+{
+    public interface IMessageBox
+    {
+        void Notify(string text);
+        void NotifyError(string text, string caption);
+        void NotifyWarn(string text, string caption);
+        bool Question(string text, string caption);
+        bool Prompt(string text, string caption);
```

Comment:
It's not clear what the intended usage of `Question` and `Prompt` are - they sounds the same to me. 

It's also not clear how `Notify` differs from `NotifyError`/`NotifyWarn`.